subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_verification.adoc

```diff
@@ -572,6 +572,10 @@ If you use git and use the binary version, make sure to make it treat this file
 ----
 *.gpg           binary
 ----
+
+[NOTE]
+Verification of signatures is disabled while running this command, similarly to `--write-verification-metadata`
```

Comment:
```suggestion
This command disables signature verification, similar to `--write-verification-metadata`.
```
Suggestion: use active voice instead of passive voice. Add a period at the end for consistency with the rest of the page.